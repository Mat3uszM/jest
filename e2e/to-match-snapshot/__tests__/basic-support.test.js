test('snapshots', () => expect({apple: "updated value"}).toMatchSnapshot());