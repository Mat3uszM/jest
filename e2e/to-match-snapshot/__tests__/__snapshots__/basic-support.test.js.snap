// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots 1`] = `
{
  "apple": "original value",
}
`;
